src/verify/VerificationStartScreen.tsx

```diff
@@ -140,6 +142,17 @@ function VerificationStartScreen({
     }
   }, [route.params?.selectedCountryCodeAlpha2])
 
+  useEffect(() => {
+    const interval = setInterval(async () => {
+      if (!signedMessageCreated) {
```

Comment:
I'm surprised this works. `signedMessageCreated` is always `false` and this hook doesn't get called when it changes.
Doesn't it continue to call `await retrieveSignedMessage()` indefinitely (until the component is unmounted)?

Reply:
yeah good callout, i missed this. i remember thinking about the outdated reference to `signedMessageCreated` for a moment but clearly never bothered to go back and test. i updated the hook here https://github.com/valora-inc/wallet/pull/3071